Week1/homework/world-queries.txt

```diff
@@ -4,7 +4,9 @@ For example:
 
 1. What are the names of all the cities in the Netherlands?
 
-SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE country.Name = 'Netherlands';
+SELECT city.Name FROM city 
```

Comment:
Looks like you also included your week1 homework in this PR. It can be a bit tricky in git to get it right, but be careful that a PR only includes the changes you want to be reviewed.